frontend/src/app/main-table/cost-table/cost-table.component.html

```diff
@@ -5,23 +5,29 @@
   </div>
 
   <p>
-    Estimated cost per day of notebook server resources – summed values may not
+    Estimated cost of notebook server resources over the last 24 hours – summed values may not
     match total due to rounding
   </p>
 
   <mat-divider></mat-divider>
-  <table style="width: auto">
+  <table *ngIf="aggregatedCost!==null && aggregatedCost.data!==undefined; else serverError" style="width: auto">
```

Comment:
Looks like `aggregatedCost` is `null` until a successful response occurs. So this would result in a failure message to the user while waiting for a response (which may succeed).